gemfiles/rails4.gemfile

```diff
@@ -8,6 +8,8 @@ gem 'eventmachine' if ENV['EM']
 gem "activesupport", "< 5.0.0"
 gem "activemodel",   "< 5.0.0"
 
+gem 'generator_spec'
```

Comment:
Should be in the 'development' group? or we want it as a real dependency?

Reply:
As far as I understood the way it work, `Gemfile` in a gem is just like `add_development_dependency` and therefore shouldn't be a real dependency.
Only the one declared in the gemspec file with `add_dependency` will be real dependency.

If that statement is wrong, then I should update it, of course!